### PR TITLE
GitHub actions feature

### LINKED ISSUE #3: [Idea] Automation of data collection
Looking at the repository and its purpose, it would be possible to set up GitHub Action based on Cron to automatically run the `update.sh` and commit the collected data directly into the repository.

Setting the Cron action to run every 48 hours or so would keep the repository reasonably up-to-date.

Is it something, that seems useful?

### PR DESCRIPTION
**Pull request by JanCaha**
Closes #3 

Moved the tool from #2 into a separate branch as requested. 

Added github action workflow to download the data automatically. The lines
`
          # git add --all
          # git commit -m "New data collected at ${{ steps.date.outputs.date }}"
`
Need to uncommented after merge. It is to avoid larger PR.

### CODE REVIEW

File: .github/workflows/run_job.yaml [hunk 1]
```diff
@@ -0,0 +1,41 @@
+name: Run data collection
+
+on:
+  push:
+    paths:
+      - '.tracked_repos'
+      - '**.R'
+      - '.github/workflows/run_job.yaml'
+  schedule:
+    - cron:  '59 23 * * 0'
```
Comment (MichaelChirico) [inline .github/workflows/run_job.yaml:10]: updated to run weekly. even monthly would probably be OK...

File: .github/workflows/run_job.yaml [hunk 2]
```diff
@@ -0,0 +1,41 @@
...
+        run: echo "::set-output name=date::$(TZ=UTC date +%Y-%m-%d_%H-%M-%S)"
+
+      - name: Commit new data
+        run: |
+          git add --all
+          git commit -m "New data collected at ${{ steps.date.outputs.date }}"
+
+
+      - name: Push data
+        uses: ad-m/github-push-action@master
```
Comment (MichaelChirico) [inline .github/workflows/run_job.yaml:38]: is there a way to make this push-to-current-branch?

I saw this but didn't know how to apply here

https://stackoverflow.com/a/58034787/3576984

**Review by MichaelChirico — APPROVED** [review]

**Comment by MichaelChirico** [comment]
Merging now 🤞

File: .github/workflows/run_job.yaml (hunk 2 above)
Comment (JanCaha) [reply .github/workflows/run_job.yaml:38]: It is a little bit more complicated but it can be done. Small PR solving this coming your way.

**Pull request merged**